superset/models/core.py

```diff
@@ -760,11 +760,11 @@ def get_perm(self) -> str:
 
     def has_table(self, table: Table) -> bool:
         engine = self.get_sqla_engine()
-        return engine.has_table(table.table_name, table.schema or None)
+        return engine.has_table(table.table_name, table.schema or None) or table.table_name in engine.table_names()
```

Comment:
I am the original Kylinpy maintainer, I just saw the source code of kylinpy. I think you should change [here](https://github.com/Kyligence/kylinpy/blob/04f360f9d07af8cf081e6d259893ba8a88b9376b/kylinpy/sqla_dialect.py#L124) .

Reply:
Table_names is the function of orm and seems effective to other db source，not only kylin. It can avoid that the engine does not implement relevant functions.

Reply:
The `has_table` is similar to `table_names` that should implement in dialect.